com.github.matfantinel.reminduck.json

```diff
@@ -0,0 +1,72 @@
+{
+    "app-id": "com.github.matfantinel.reminduck",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "command": "com.github.matfantinel.reminduck",
+    "finish-args": [
+       "--share=ipc",
+       "--share=network",
+       "--socket=x11",
+       "--socket=wayland",
+       "--talk-name=org.gtk.vfs.*",
+       "--talk-name=ca.desrt.dconf",
+       "--filesystem=home",
+       "--filesystem=xdg-run/dconf",
+       "--filesystem=~/.config/dconf:ro",
+       "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "cleanup": [
+       "/include",
+       "/lib/pkgconfig",
+       "/man",
+       "/share/doc",
+       "/share/gtk-doc",
+       "/share/man",
+       "/share/pkgconfig",
+       "/share/vala",
+       "*.la",
+       "*.a"
+    ],
+    "modules": [
+       {
+          "name": "libyaml",
+          "sources": [
+              {
+                  "type": "git",
+                  "url": "https://github.com/yaml/libyaml.git",
```

Comment:
Use a tarball or add a commit along with the tag. The same for the other modules.

Reply:
Please don't mark conversations as resolved unless they are.